### PR TITLE
Fix incorrect execaNode signature in `index.d.ts`

### PR DESCRIPTION
**Pull request by forivall**
All signatures of `execaNode` support `nodeOptions` (and `nodePath`)

### CODE REVIEW

**Comment by sindresorhus** [comment]
CI is failing

File: index.test-d.ts [hunk 1]
```diff
@@ -210,3 +210,7 @@ expectType<ExecaReturnValue>(
 expectType<ExecaReturnValue<Buffer>>(
 	await execaNode('unicorns', ['foo'], {encoding: null}),
 );
+execaNode('unicorns', {nodeOptions: ['--async-stack-traces']})
```
Comment (ehmicky) [inline index.test-d.ts:213]:
```ts
execaNode('unicorns', {nodeOptions: ['--async-stack-traces']});
```

**Comment by sindresorhus** [comment]
Bump

**Comment by forivall** [comment]
thanks for the bump! looking now

**Review by ehmicky — APPROVED** [review]
Thanks @forivall!

**Comment by ehmicky** [comment]
@sindresorhus Is this ok to merge?

**Pull request merged**